language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/components/tests/helpers/DummyLedgerClient.scala

```diff
@@ -36,12 +36,11 @@ class DummyLedgerClient(
         verbose: Boolean): Flowable[GetActiveContractsResponse] =
       activeContractSet
 
-    // TODO Untested endpoint
     override def getActiveContracts(
         filter: TransactionFilter,
         verbose: Boolean,
         accessToken: String): Flowable[GetActiveContractsResponse] =
-      ???
+      untestedEndpoint
```

Comment:
Great way to improve readability!

Reply:
This was a suggestion by @SamirTalwar. :slightly_smiling_face: